_pages/resources.html

```diff
@@ -21,8 +21,8 @@ <h5>2018 FRC Workshop Resources</h5>
     <p><a href="assets/resources/workshops/2018/Project-Management.pdf">Project Management Presentation</a></p>
     <p><a href="assets/resources/workshops/2018/Vision-Processing.pdf">Vision Processing Slides</a></p>
     <p><a href="assets/resources/workshops/2018/3D-Printing.pdf">3D Printing</a></p>
-    <p><a href="assets/resources/workshops/2018/Orthographic-and-Isometric-Projections.pdf">Orthographic and Isometric Projections</a></p>
-    <p><a href="assets/resources/workshops/2018/C++Java.pdf">C++ and Java Presentation</a></p>
+    <p><a href="assets/resources/workshops/2018/Orthographic-and-Isometeric-Projections.pdf">Orthographic and Isometric Projections</a></p>
```

Comment:
How about you change the filename to the correct spelling rather than using the wrong spelling again here?